ci/pipeline.properties

```diff
@@ -14,6 +14,7 @@ docker.mongodb.7.0.version=7.0.2
 
 # Supported versions of Redis
 docker.redis.6.version=6.2.13
+docker.redis.7.version=7.2.4
```

Comment:
Mind updating the property in Spring Data Build as well?

Reply:
created the task already spring-projects/spring-data-build#2212 waiting for final review.